src/helpers/asset-bundles.ts

```diff
@@ -2,46 +2,46 @@ import { DeploymentToSqs } from '@dcl/schemas/dist/misc/deployments-to-sqs'
 import { fetch } from 'undici'
 import { CliError } from '../bin'
 
-export const multiPlatformFlag = 'MultiPlatform'
+export const defaultAbAdmin = 'https://ab-admin.decentraland.org'
```

Comment:
NIT: to state that the default server is the production one
```suggestion
export const productionAbAdmin = 'https://ab-admin.decentraland.org'
```